ewitem.py

```diff
@@ -1294,7 +1295,7 @@ def find_poudrin(id_user = None, id_server = None):
 			item_data.item_props["adorned"] = 'false'
 			item_data.persist()
 
-		if item_sought.get('soulbound'):
+		if item_sought.get('soulbound') and EwItem(id_item = item_sought.get('id_item')).item_props["context"] != "housekey":
```

Comment:
this is a weird special case. why do house keys need to be soulbound in the first place?

Reply:
also item_props["context"] will throw an error if the item has no "context" prop. use get("context") instead

Reply:
housekeys shouldn't be able to be destroyed or dropped on death. it would've required way more work to cover my bases there, so i made them soulbound with exceptions instead. besides, the general consensus was that people preferred them to have soulbound-like properties.

Reply:
changed the ["context"] bit though

Reply:
it'd be funnier if they were able to be destroyed or dropped on death imo. just check if the item still exists in the methods that reference the keys. non-existent items should return with an item id of -1 iirc.

if you feel strongly about this, it's fine though.

Reply:
it'd be funnier if they were able to be destroyed or dropped on death imo. just check if the item still exists in the methods that reference the keys. non-existent items should return with an item id of -1 iirc.

if you feel strongly about this, it's fine though.

Reply:
i could see the value of being able to destroy it, so i'll look into the possibility of being able to recycle or cliff them. i don't want to have them drop on death, though. that would remove their purpose, and they wouldn't be worth the price of admission. if you push the update now i'll patch that in later.

Reply:
think of the gameplay possibilites though. stalking and killing players for the keys they possess. frantically messaging your friend to change their locks after losing the key to their house. besides you don't drop all of your items on death, and with the pvp flag system you have a large amount of control over when you are vulnerable.

patching it in a future update sounds fine, but if we decide to make housekeys not soulbound anymore, it'll mean I have to go into the database and change the existing ones manually

Reply:
this is a tough decision. i still think we should err on the side of caution though. when i made it possible to steal from apartments, it resulted in nobody inviting anyone over because they were afraid of getting robbed. i think this suggestion would do the same thing for housekeys. and once the novelty wore off it would just be really inconvenient.  besides, isn't the query for that pretty easy?

update rfck.items_prop ip inner join rfck.items it
on ip.id_item = it.id_item
set it.soulbound = 0
where name = 'id_item' and value = 'housekey'

i changed my mind on making it droppable later. i'm gonna do it right now.